public/editor/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -15,6 +15,13 @@ define(function(require) {
         // Determine where to horizontally place menu based on button's icon location
         var menuWidth = self._menu$.width();
         var leftOffset = self._button$.offset().left - menuWidth/2 + 11;
+        var iconWidth = $("#filetree-pane-nav-add").width();
```

Comment:
Because this code is used for both of the file menu and the editor gear menu, we can't just reference the ``filetree-pane-nav-add`` element.

Instead, you can use   ``var iconWidth = self._button$.width();`` - the $button element is the icon that is clicked to open both menus.